### PR TITLE
NF: replace requireNonNull by !!

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Ideally we should lint against it, but I can't imagine someone introducing it.

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.kt [hunk 1]
```diff
@@ -35,7 +35,10 @@ import com.ichi2.libanki.Collection.CheckDatabaseResult
 import com.ichi2.libanki.importer.AnkiPackageImporter
 import com.ichi2.libanki.sched.DeckDueTreeNode
 import com.ichi2.libanki.sched.TreeNode
-import com.ichi2.utils.*
+import com.ichi2.utils.Computation
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.kt:38]: ..

**Review by david-allison — DISMISSED** [review]
one nitpick to not change imports unnecessarily. Change looks great, thanks!

File: AnkiDroid/src/test/java/com/ichi2/anki/dialogs/utils/RecursivePictureMenuUtil.kt [hunk 2]
```diff
@@ -26,7 +26,7 @@ class RecursivePictureMenuUtil {
     companion object {
         @CheckResult
         fun getRecyclerViewFor(menu: RecursivePictureMenu): RecyclerView {
-            val dialog = Objects.requireNonNull(menu.dialog) as MaterialDialog
+            val dialog = menu.dialog!! as MaterialDialog
```
Comment (oakkitten) [inline AnkiDroid/src/test/java/com/ichi2/anki/dialogs/utils/RecursivePictureMenuUtil.kt:29]: I think you can just say `menu.dialog as MaterialDialog` no? You should get a slightly better NPE

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.kt (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.kt:38]: I still have no idea how to let Android Studio not do any changes. The options I have got are either: no wildcard. Or using wild card after N import. By default N being 5. So I guess in this case it detected only four elements were detected. I searched online but could not find any way not to select any of the options of this image.
2022-10-02-160451_1916x1040_scrot

**Review by lukstbit — APPROVED** [review]
LGTM
I'll leave this on "Pending merge" for David's raised issue.

**Comment by Arthur-Milchior** [comment]
Done

File: AnkiDroid/src/test/java/com/ichi2/anki/dialogs/utils/RecursivePictureMenuUtil.kt (hunk 2 above)
Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/anki/dialogs/utils/RecursivePictureMenuUtil.kt:29]: This still seems relevant

**Review by david-allison — COMMENTED** [review]
I'm fine to merge, one comment to address

File: AnkiDroid/src/test/java/com/ichi2/anki/dialogs/utils/RecursivePictureMenuUtil.kt (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/test/java/com/ichi2/anki/dialogs/utils/RecursivePictureMenuUtil.kt]: Thanks. Done

**Review by BrayanDSO — APPROVED** [review]

**Pull request merged**